README.md

```diff
@@ -27,7 +27,7 @@ Steerage.init({ config: Path.join(__dirname, 'config', 'config.json') }).then((s
 
 ### Configuration options
 
-- `config` - a fully resolved path to a configuration document (relative paths in this document are from the document's location).
+- `config` - String or array of fully resolved paths to configuration documents. Defaults to `[]`. (relative paths in this document are from the document's location).
```

Comment:
I think you need to [require](https://github.com/ExpediaGroup/steerage/pull/23/files#diff-6d186b954a58d5bb740f73d84fe39073R38) this option...as in not defaulting to `[]`. Otherwise, `mergeManifestConfigs()` would resolve to undefined.